content/articles/two-factor-authentication-enforcement.markdown

```diff
@@ -0,0 +1,78 @@
+---
+title: Enforce Two-Factor Authentication for a shared Account
+excerpt: How to protect your shared DNSimple account using Two-Factor authentication enforcement.
+categories:
+- Account
+---
+
+# Enforce Two-Factor Authentication for a shared Account
+
+<info>
+This article describes a feature in Public Beta.
+</info>
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+DNSimple supports [Two-Factor authentication](/articles/two-factor-authentication/). This feature enforces that all users in our account must have Two-Factor authentication enabled.
```

Comment:
```suggestion
[Two-Factor authentication](/articles/two-factor-authentication/) provides an extra level of security to protect your user credentials. You can enforce account members to use two-factor authentication by enabling the feature for the account.
```

I replaced the concept of _your_ account as it seems you own an account. Whereas, in fact, the account is not owned by anyone, but members belong to an account.